test/rekt/README.md

```diff
@@ -14,3 +14,34 @@ To run just one test:
 ```bash
 SYSTEM_NAMESPACE=knative-eventing go test -count=1 -v -tags=e2e -run Smoke_PingSource ./test/rekt/...
 ```
+
+## Broker tests.
+
+The Broker class can be overridden by using the envvar `BROKER_CLASS`. By default, this will be `MTChannelBasedBroker`. 
+
+The Broker templates can be overridden by using the env var `BROKER_TEMPLATES`. 
+
+```bash
+BROKER_CLASS=MyCustomBroker
+BROKER_TEMPLATES=/path/to/custom/templates
+SYSTEM_NAMESPACE=knative-eventing \
+  go test -count=1 -v -tags=e2e -run Smoke_PingSource ./test/rekt/...
+```
+
+### Custom templates
+
+The minimum shape of a custom template for namespaced resources:
+
+```yaml
+apiVersion: rando.api/v1
+kind: MyResource
+metadata:
```

Comment:
Since the only thing we have here is Broker, I'dd add this as an example:
```
  {{ if .brokerClass }}
  annotations:
    eventing.knative.dev/broker.class: {{ .brokerClass }}
  {{ end }}
```

Reply:
ack, but maybe we followup with broker specific docs when this lands and we get it going in other places ? I am showing yaml for not a broker